NitroxServer/Communication/Packets/Processors/FootstepPacketProcessor.cs

```diff
@@ -0,0 +1,52 @@
+using NitroxModel.DataStructures.Unity;
+using NitroxModel.Packets;
+using NitroxServer.Communication.Packets.Processors.Abstract;
+using NitroxServer.GameLogic;
+
+namespace NitroxServer.Communication.Packets.Processors
+{
+    public class FootstepPacketProcessor : AuthenticatedPacketProcessor<FootstepPacket>
+    {
+        private readonly float footstepAudioRange = 20f;
+        private readonly PlayerManager playerManager;
+
+        public FootstepPacketProcessor(PlayerManager playerManager)
+        {
+            this.playerManager = playerManager;
+        }
+
+        public override void Process(FootstepPacket footstepPacket, Player sendingPlayer)
+        {
+            var players = playerManager.GetAllPlayers();
+            foreach (Player player in players)
+            {
+                if (sendingPlayer.SubRootId.HasValue)
+                {
+                    if (player.SubRootId.HasValue)
+                    {
+                        // If both players have id's, check if they are the same
+                        if (NitroxVector3.Distance(player.Position, sendingPlayer.Position) <= footstepAudioRange && player != sendingPlayer && player.SubRootId.Value == sendingPlayer.SubRootId.Value)
```

Comment:
Line is too long imo
```suggestion
                        if (NitroxVector3.Distance(player.Position, sendingPlayer.Position) <= footstepAudioRange &&
                            player != sendingPlayer && player.SubRootId.Value == sendingPlayer.SubRootId.Value)
```